### PR TITLE
dpdata

### PR DESCRIPTION
**Pull request by njzjz**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by njzjz** [comment]
@conda-forge/python Please review. Thank you.

File: recipes/dpdata/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{% set name = "dpdata" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "53164fd67c5ba2bf2456969eb80adc85c2ca7768957c58716b4b00934de233db" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/py3/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-py3-none-any.whl
```
Comment (xhochy) [inline recipes/dpdata/meta.yaml]: We prefer to build from the sdist and not the wheel.

File: recipes/dpdata/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  sha256: {{ sha256 }}
```
Comment (xhochy) [inline recipes/dpdata/meta.yaml]: Just write the sha256 here instead of making it a parameter.

File: recipes/dpdata/meta.yaml (hunk 1 above)
Comment (njzjz) [reply recipes/dpdata/meta.yaml]: If so, I need to add a patch because the source code do not contain the version number.

Comment (xhochy) [reply recipes/dpdata/meta.yaml]: Thanks, I gave upstream a hint on how to fix this easily: https://github.com/deepmodeling/dpdata/issues/18

**Pull request merged**